data/mods/gen1/rulesets.ts

```diff
@@ -44,7 +45,7 @@ export const Rulesets: {[k: string]: ModdedFormatData} = {
 				spe: newSpecies.baseStats.hp,
 			};
 			for (const i in newSpecies.baseStats) {
-				newSpecies.baseStats[i] = stats[i];
+				newSpecies.baseStats[i as StatID] = stats[i as StatID];
```

Comment:
I think I prefer the type declaring above the `for` loop compared to typecasting.

Reply:
in latest rev